### ISSUE TITLE
How do I preserve text selection when dragging? 

### ISSUE THREAD

**Issue by johndevor** [react-grid-layout/react-draggable#384]
When I initiate a drag, any text selected in `<div>...</div>` is deselected. How do I avoid this?

```
import React, { Component } from 'react'
import Draggable from 'react-draggable'


export default class Window extends Component {
  render() {
    return (
      <Draggable>
        <div>
          <p><u>Title</u></p>
          {this.props.children}
        </div>
      </Draggable>
    )
  }
}
```

**Comment by grg-akshay** [comment]
You may pass a prop `enableUserSelectHack={false}` to Draggable

**Comment by johndevor** [comment]
Hmmm, that didn't do the trick by itself, but I was able to get it working by adding onMouseDown={e => e.preventDefault()} to the handle element like so:

```
<Draggable handle="#handle" enableUserSelectHack={false}>
  <WindowContainer>
    <div id="handle">
      <WindowTitleBar onMouseDown={e => e.preventDefault()}>
        My Title
      </WindowTitleBar>
    </div>
    {this.props.children}
  </WindowContainer>
</Draggable>
```

Thanks!

**Comment by STRML** [comment]
This is fixed in #470

**Comment by emaborsa** [comment]
Doesn't work for me. I used `cancel={"input, svg, button, table"}` in order to prevent dragging from input fields, images, buttons and tables, but even if I added `enableUserSelectHack={false}`, the text in tables are not selectable.

```import { FC } from "react";
import ReactDraggable from "react-draggable";
import styles from "./Draggable.module.css";

export const Draggable: FC = ({ children }) => {
  return (
    <ReactDraggable cancel={"input, svg, button, table"} enableUserSelectHack={false}>
      <div className={styles.draggable}>{children}</div>
    </ReactDraggable>
  );
};```
